applications/ChimeraApplication/custom_processes/apply_chimera_process_monolithic.h

```diff
@@ -0,0 +1,182 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+// ==============================================================================
+//  ChimeraApplication
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Authors:        Aditya Ghantasala, https://github.com/adityaghantasala
+// 					Navaneeth K Narayanan
+//					Rishith Ellath Meethal
+// ==============================================================================
+//
+
+#if !defined(KRATOS_APPLY_CHIMERA_MONOLITHIC_H_INCLUDED)
+#define KRATOS_APPLY_CHIMERA_MONOLITHIC_H_INCLUDED
+
+// System includes
+#include "omp.h"
```

Comment:
```suggestion

```

seems unused and should not be directly included, otherwise won't compile without OpenMP